app/src/main/java/com/example/benchproject/app/common/binding/chip/ChipBindingAdapters.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.benchproject.app.common.binding.chip
+
+import androidx.databinding.BindingAdapter
+import com.example.benchproject.app.movie.details.entity.Genres
+import com.google.android.material.chip.Chip
+import com.google.android.material.chip.ChipGroup
+
+@BindingAdapter("addGenresToChips")
```

Comment:
Try to follow xml attributes naming, in this case eg "genresChips"

Reply:
Ok, I changed it